colcon-build/README.md

```diff
@@ -0,0 +1,42 @@
+# colcon-build
```

Comment:
Can you mark `colcon-build-and-test` as `deprecated`?
It's better if we can print some warnings when the users run the action.

Reply:
How do I mark it?
Add the word `DEPRECATED` to the README.md?

```
# DEPRECATED: colcon-build-and-test
```

Reply:
Yes, it's the minimum requirement.
In addition, I'd like to add `echo ::warning::` or something like that in the action steps.

Could you search [this document](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions) for the best way?

Reply:
I added deprecated warnings in the workflow.
How do you feel this message? Is the message clear?

https://github.com/autowarefoundation/autoware-github-actions/actions/runs/1794329134

Reply:
So nice! Thank you. :pray: